sticht/slack.py

```diff
@@ -176,16 +203,23 @@ def get_summary_blocks_for_deployment(self) -> List[Dict]:
                     'elements': button_elements,
                 },
             )
+
+        # If blocks is going to be longer than 3kB, trim summary_parts. This should help avoid invalid_blocks error.
+        json_blocks = json.dumps(blocks)
+        if len(json_blocks) > 2900:
+            excess_length = len(json_blocks) - 2900
+            blocks[0]['text']['text'] = blocks[0]['text']['text'][:-excess_length]
```

Comment:
Are you sure `[:2900]` isn't simpler than calculating the excess?

Reply:
oh .... d'oh, the limit is 3000 for the text field itself, not the whole JSON object. I'll simplify.